src/extra/JsonSchemaRenderer/types.ts

```diff
@@ -0,0 +1,36 @@
+import { FormatValidator, FormatDefinition } from 'ajv';
+export { JSONSchema7 as JSONSchema } from 'json-schema';
+
+export const JsonTypes = {
+	array: 'array',
+	object: 'object',
+	number: 'number',
+	integer: 'integer',
+	string: 'string',
+	boolean: 'boolean',
+	null: 'null',
+};
+
+export type DefinedValue =
+	| number
+	| string
+	| boolean
+	| { [key: string]: any }
+	| any[];
+
+export type Value = DefinedValue | null;
+
+export interface UiSchema {
```

Comment:
Is this the same with the Form's uiSchema, or at least a subset? If that's the case, let's use `Pick` or `Omit` instead, otherwise it's fine